holoviews/plotting/mpl/comms.py

```diff
@@ -0,0 +1,89 @@
+import uuid
+import warnings
+
+try:
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        from matplotlib.backends.backend_nbagg import CommSocket, new_figure_manager_given_figure
+except ImportError:
+    CommSocket = object
+from mpl_toolkits.mplot3d import Axes3D
+
+from ..comms import JupyterComm
+
+mpl_msg_handler = """
+target = $('#{comms_target}');
+img = $('<div />').html(data);
+target.children().each(function () {{ $(this).remove() }})
+target.append(img)
+"""
```

Comment:
Given this is the body of the function and not the complete function, I think a comment could help make it clearer. E.g:

mpl_msg_handler = """
/* Body of the msg_handler function that is passed the message */
target = $('#{comms_target}');
img = $('<div />').html(data);
target.children().each(function () {{ $(this).remove() }})
target.append(img)
"""

I think it should be Javascript comment as it is about Javascript code. I think this will help anyone copying and pasting this to write a new message handler.